DOL.WHD.Section14c.PdfAPi/PdfHelper/PdfHelper.cs

```diff
@@ -135,23 +142,26 @@ private static PdfDocument ConcatenatePDFDocumentByBytes(PdfDocument outputDocum
         /// <param name="outputDocument"></param>
         /// <param name="buffer"></param>
         /// <returns></returns>
-        private static PdfDocument ConcatenatePDFWithImage(PdfDocument outputDocument, byte[] buffer)
+        private static PdfDocument ConcatenatePDFWithImage(byte[] buffer, string fileName)
```

Comment:
Should this method be renamed now?  Maybe something like `CreatePDFWithImage` since it creates a new PDF document?